resources/views/members/index.blade.php

```diff
@@ -130,43 +135,43 @@
 						<td>{{ $member->telefoon }}</td>
 						<td><a href="mailto:{{ $member->email }}">{{ $member->email }}</a></td>
 					</tr>
-				@endforeach
-			</tbody>
-		</table>
+					@endforeach
+				</tbody>
+			</table>
+		</div>
+		@endcan
 	</div>
-	@endcan
-  </div>
 
 </div>
 @endsection
 
 @section('footer')
 <!-- These scripts load DataTables -->
 <script type="text/javascript">
-$( document ).ready(function() {
+	$( document ).ready(function() {
+
+	// Create the column options for the members table (number of columns can vary with user capabilities)
+	let memberColumnCount = $('#currentMembersTable').find('tr:first th').length;
+	let orderableColumnIndices = [0, 2, 3];
+	let columnOptions = [];
+	for (let i = 0; i < memberColumnCount; i++) {
+		let option = (orderableColumnIndices.includes(i)) ? null : {'orderable': false};
+		columnOptions.push(option);
+	}
+
     $('#currentMembersTable').DataTable({
 		paging: false,
 		order: [[ 0, "asc" ]],
-		columns: [
-			null,
-			{'orderable':false},
-			null,
-			null,
-			null,
-			null,
-			{'orderable':false},
-			{'orderable':false}
-		]
+		columns: columnOptions
 	});
-	
+
 	$('#formerMembersTable').DataTable({
 		order: [[ 0, "asc" ]],
 		columns: [
 			null,
 			{'orderable':false},
 			null,
 			null,
-			//null,
 			{'orderable':false},
 			{'orderable':false}
```

Comment:
Misschien ter standardizatie hier hetzelfde doen als bij de andere tabel. Mochten we daar de kolomen verandere / dynamisch maken dan blijft dit ook werken.